pages/api/notes/index.ts

```diff
@@ -1,25 +1,70 @@
 import { NextApiRequest, NextApiResponse } from 'next'
 import fs from 'fs';
 
+import Database from 'better-sqlite3';
+
+const options = {};
+export const db = new Database('foobar.db', options);
+
+function tables() {
+  const stmt = db.prepare(`create table if not exists notes (
+    id integer primary key autoincrement,
+    text text not null,
+    time real not null,
+    video_id text not null,
+    run_id text
+);`);
+  stmt.run([]);
+  const notes = size_notes();
+  if (notes.size == 0) {
+    initial_data();
+  }
+}
+
+export function notes_read(id: number) {
```

Comment:
this is a fine first iteration. for the future, moving most of this logic to e.g. `collections/Notes.ts` and `models/Note.ts` is a more scalable design pattern.

Reply:
Moved database functions to `collections/Notes.ts`.  Added a `models/Note.ts` with a `Note` type.